pkgs/top-level/python-packages.nix

```diff
@@ -7535,8 +7535,6 @@ in {
 
   selectors2 = callPackage ../development/python-modules/selectors2 { };
 
-  selectors34 = callPackage ../development/python-modules/selectors34 { };
```

Comment:
Please add an entry to python-aliases.

Reply:
@SuperSandro2000 I'm not sure what should be aliased here.. Selectors2 is a different project and also archived.

Reply:
Please add a throw alias to python-aliases.nix.

Reply:
Sorry, I'm still not sure what you are requesting here .. we usually don't put notes on removed python packages as we do with with application packages.

Reply:
We are doing this since recently because before the addition of python-aliases.nix it was not easily possible.

Reply:
all throws seems to live in python-packages.nix, so I put mine there, too.

Reply:
Thats the old functionality. The need to be carried over to python-aliases. Please add yours there, too.

Reply:
@SuperSandro2000  I think I got it right now.